File: ConsoleApp1/Timeser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp1
{
    class Multiplier : IProcess
    {
        public Multiplier() { }
        public dynamic Process(dynamic A, dynamic B)
        {
            return A * B;
        }
        public int ModProcess(int A, int B, int M)
        {
            return (A * B) % M;
        }
    }
}

File: src/com/company/Timeser.java
package com.company;

public class Timeser implements Process {
    public Timeser() {
    }

    @Override
    public double process(double a, double b) {
        return a * b;
    }

    @Override
    public int modProcess(int a, int b, int m) {
        return (a * b) % m;
    }
}
